rcjaRegistration/templates/common/loggedInbase.html

```diff
@@ -163,7 +131,10 @@
                         <a href="{% url 'users:details' %}" class="item">My details</a>
                         {% if user.currentlySelectedSchool %}<a href="{% url 'schools:details' %}" class="item">School details</a>{% endif %}
                         <a href = "{% url 'password_change' %}" class="item">Change password</a>
-                        <a href = "{% url 'logout' %}" class="item">Log out</a>
+                        <form method="post" action="{% url 'logout' %}" class="button-form">
+                            {% csrf_token %}
+                            <input class="header-button item" type="submit" value="Log out"/>
```

Comment:
@ZMcWilliam because Django no longer supports GET methods for logout I have needed to make these changes. However the CSS class seems wrong because in the dropdown, this item is slightly lighter than the others on mouseover.  Could you please take a look at this because you CSS is much better than mine.

Reply:
Moved to #721